### ISSUE TITLE
This tool has moved to UMEP for processing, available via the QGIS Plugin Manager. Visit our online manual for more information.

### ISSUE THREAD

**Issue by kun-creator** [UMEP-dev/UMEP#698]
![Image](https://github.com/user-attachments/assets/fb4e1a46-e686-42a3-bc95-ce24e8e35f41)
I've already installed UMEP for processing.But when I tried to use Sky View Factor,it was wrong.

**Comment by biglimp** [comment]
Yes, so you need to open the tool from the Processing toolbox in QGIS.
